viam-cartographer.go

```diff
@@ -222,7 +222,7 @@ type cartographerService struct {
 
 // GetPosition forwards the request for positional data to the slam library's gRPC service. Once a response is received,
 // it is unpacked into a Pose and a component reference string.
-func (cartoSvc *cartographerService) GetPosition(ctx context.Context, name string) (spatialmath.Pose, string, error) {
+func (cartoSvc *cartographerService) GetPositionOld(ctx context.Context, name string) (spatialmath.Pose, string, error) {
```

Comment:
Not sure of the best way to implement this, another possibility is to remove GetPositionOld entirely, as nothing should use it.

Reply:
I think this can indeed be removed. Nothing calls it & adding this method which is never & will never be called will only confuse.

Reply:
[q] Also, is there any reason to keep `GetPointCloudMapStream` & `GetInternalStateStream` within viam-cartogropher.go?

Reply:
yes, because `GetPointCloudMapStream` & `GetInternalStateStream` are required to fulfill the slam service interface still

Reply:
Understood!